CHANGELOG.rst

```diff
@@ -4,6 +4,23 @@ Ansible Playbooks for Confluent Platform - Release Notes
 
 .. contents:: Topics
 
+v7.6.0
+======
+
+New features
+-------------
+
+- In place upgrade from Zookeeper to Kraft is now supported
+- ARM64 architecture is now supported
+
+Notable enhancements
+-------------
+
+- Oracle8, Oracle9, Rocky8, Rock9 Operating Systems are now supported
+- Discovery is now a separate repo named cp-discovery
```

Comment:
Discovery has be de-coupled from cp-ansible repo and is now present in a new repo named cp-discovery